package/python-kiwi-spec-template

```diff
@@ -212,12 +212,12 @@ BuildRequires:  dracut
 Requires:       bc
 Requires:       cryptsetup
 %if 0%{?fedora} || 0%{?rhel} || 0%{?ubuntu} >= 1804 || 0%{?debian} >= 9
-Requires:       btrfs-progs
+Recommends:     btrfs-progs
```

Comment:
Actually, we can't use `Recommends` on EL7, so this needs to be done slightly more carefully...

Reply:
Suggestion:

```rpm-spec
%if 0%{?rhel} && 0%{?rhel} < 8
Requires:       btrfs-progs
%else
Recommends:     btrfs-progs
%endif
```

Reply:
ah yes, I always forgot to remind me on that. Thanks and updated